shub/config.py

```diff
@@ -81,11 +82,12 @@ def load(self, stream):
                 yaml_option_conf = yaml_cfg.get(option, {})
                 option_conf.update(yaml_option_conf)
                 if option == 'images' and yaml_option_conf:
-                    click.echo(
+                    warnings.warn(
```

Comment:
I'm not sure if I like this. It spams the argument list of all commands for an edge case, and the time saved is negligible:

```
jakob@MosEisley ~/shub/playground % python -m timeit -s "from shub.config import load_shub_config" "load_shub_config()" 
1000 loops, best of 3: 1.61 msec per loop
```